pkg/router/seller.go

```diff
@@ -64,12 +76,37 @@ func sellerEditInfo(pg *db.DB, logger *zap.SugaredLogger) echo.HandlerFunc {
 			logger.Error(err)
 			return echo.NewHTTPError(http.StatusBadRequest)
 		}
+		fileHeader, err := c.FormFile("image")
+		if err == nil {
+			imageID, err := mc.PutFile(c.Request().Context(), fileHeader, common.GetFileName(fileHeader))
+			if err != nil {
+				logger.Error(err)
+				return echo.NewHTTPError(http.StatusInternalServerError)
+			}
+			param.ImageID = imageID
+		} else if err.Error() == "http: no such file" {
```

Comment:
how did this come from, consider set as const string

Reply:
```suggestion
		} else if errors.Is(err, http.ErrMissingFile) {
```